src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -1,249 +1,190 @@
-                   type="text"
-                   placeholder="Full name..."
-                   id="name"
-                   name="name"
-                   formControlName="name">
-          </div>
-        </div>
-
-        <!-- Email -->
-        <div class="form-group row">
-          <label i18n
-                 class="cd-col-form-label"
-                 for="email">Email</label>
-          <div class="cd-col-form-input">
-            <input class="form-control"
-                   type="email"
-                   placeholder="Email..."
-                   id="email"
-                   name="email"
-                   formControlName="email">
-
-            <span class="invalid-feedback"
-                  *ngIf="userForm.showError('email', formDir, 'email')"
-                  i18n>Invalid email.</span>
-          </div>
-        </div>
-
-        <!-- Roles -->
-        <div class="form-group row">
-          <label class="cd-col-form-label"
-                 i18n>Roles</label>
-          <div class="cd-col-form-input">
-            <span class="no-border full-height"
-                  *ngIf="allRoles">
-              <cd-select-badges [data]="userForm.controls.roles.value"
-                                [options]="allRoles"
-                                [messages]="messages"></cd-select-badges>
-            </span>
-          </div>
-        </div>
-
-        <!-- Enabled -->
-        <div class="form-group row"
-             *ngIf="!isCurrentUser()">
-          <div class="cd-col-form-offset">
-            <div class="custom-control custom-checkbox">
-              <input type="checkbox"
-                     class="custom-control-input"
-                     id="enabled"
-                     name="enabled"
-                     formControlName="enabled">
-              <label class="custom-control-label"
-                     for="enabled"
-                     i18n>Enabled</label>
-            </div>
-          </div>
-        </div>
-
-        <!-- Force change password -->
-        <div class="form-group row"
-             *ngIf="!isCurrentUser() && !authStorageService.isSSO()">
-          <div class="cd-col-form-offset">
-            <div class="custom-control custom-checkbox">
-              <input type="checkbox"
-                     class="custom-control-input"
-                     id="pwdUpdateRequired"
-                     name="pwdUpdateRequired"
-                     formControlName="pwdUpdateRequired">
-              <label class="custom-control-label"
-                     for="pwdUpdateRequired"
-                     i18n>User must change password at next logon</label>
-            </div>
-          </div>
-        </div>
+<div cdsCol
+     [columnNumbers]="{md: 4}">
+  <ng-container *cdFormLoading="loading">
+    <form #frm="ngForm"
+          #formDir="ngForm"
+          [formGroup]="userForm"
+          novalidate>
+      <div i18n="form title|Example: Create Volume@@formTitle"
```

Comment:
`Create Volume@@formTitle` not getting what is this used for?

Reply:
done